### PR TITLE
Tomorrow forecast UI

### PR DESCRIPTION
**Pull request by AlexeyAnurin**
screen1
screen2

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -18,6 +18,9 @@
     <string name="today_tab">Today</string>
     <string name="tomorrow_tab">Tomorrow</string>
     <string name="week_tab">Week</string>
+    <string name="tomorrow_atmosphere_characteristics_title">Detalis</string>
+    <string name="tomorrow_humidity">Humidity</string>
```
Comment (sin-al-vl) [inline app/src/main/res/values/strings.xml:22]: Why do we need this? We already have humidity string res. It's interesting, why did you renamed `hourly_temperature_celsius_pattern` to more widely applicable name, but didn't do this to `humidity`?
I'd remove `current` prefix from all string names, except `current_atmosphere_characteristics_title', as well as remove `tomorrow` prefix from the string below.

**Review by sin-al-vl — APPROVED** [review]

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -10,7 +10,8 @@
     <color name="gray1">#808080</color>
     <color name="gray2">#6e6e6e</color>
     <color name="light_gray">#d3d3d3</color>
-    <color name="background_fragment">#27a5cf</color>
-    <color name="background_tab_layout">#2789aa</color>
-    <color name="hourly_weather_list_forecast_background">#AA9493</color>
+    <color name="hourly_weather_list_forecast_background">#3d91af</color>
+    <color name="background_fragment">#4db8dd</color>
+    <color name="background_tab_layout">#3d91af</color>
```
Comment (sin-al-vl) [inline app/src/main/res/values/colors.xml:15]: You may reuse colors here. Create color res with direct name, like
`<color name="cyan">#3d91af</color>`
or any other suitable color name. And then you may use it here and in 13th line via `@color/cyan`

**Review by sin-al-vl — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (AlexeyAnurin) [reply app/src/main/res/values/colors.xml]: fixed

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (AlexeyAnurin) [reply app/src/main/res/values/strings.xml]: fixed

**Review by sin-al-vl — APPROVED** [review]

**Pull request merged**